src/table/FieldBase.cc

```diff
@@ -20,6 +20,10 @@ template <typename T>
 struct TypeTraits;
 
 template <>
+struct TypeTraits<std::uint8_t> {
+    static char const *getName() { return "B"; }
```

Comment:
Ok, so we have one. Then why don't we use it everywhere? Preferably pulling it up to `utils`?
In that case I would still modify it slightly such that it can be used in `constexpr` contexts (e.g. one can use it in a `switch` or something).

Reply:
These characters are FITS table characters. I'm not sure I'd want to use it anywhere else.

I don't understand your suggested modification but respectfully suggest that any significant refactoring should go into a new ticket.